perf/perf_top.py

```diff
@@ -56,12 +61,54 @@ def setUp(self):
             if not smm.check_installed(package) and not smm.install(package):
                 self.cancel('%s is needed for the test to be run' % package)
 
+        # Creating a temprory file
+        self.temp_file = tempfile.NamedTemporaryFile().name
+
         # Getting the parameters from yaml file
         self.option = self.params.get('option', default='')
 
         # Clear the dmesg by that we can capture delta at the end of the test
         dmesg.clear_dmesg()
 
+    def test_workload(self):
+        """
+        Get ebizzy workload as a tar file and build it.
+        """
+        url = 'https://sourceforge.net/projects/ebizzy/files/ebizzy/0.3/ebizzy-0.3.tar.gz'
+        tarball = self.fetch_asset(url, expire='7d')
+        archive.extract(tarball, self.workdir)
+        version = os.path.basename(tarball.split('.tar.')[0])
+        self.sourcedir = os.path.join(self.workdir, version)
+        os.chdir(self.sourcedir)
+        process.run("./configure")
+        build.make(self.sourcedir, extra_args='LDFLAGS=-static')
```

Comment:
As discussed yesterday, we can still add this as a scenario in a existing cfg, making sure ebizzy is modified to run with any other tests as this is a workload